index.html

```diff
@@ -6,388 +6,492 @@
+          <li class="header__nav-item">
+            <a class="header__nav-link header__nav-link--selected" href="#">Home</a> 
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link href="#">Blog</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link href="#">Catalog</a>
+          </li>
+          <li class="header__nav-item">
+            <a class="header__nav-link href="#">Contact us</a>
+          </li>
         </ul>
       </nav>
-      <a href="#">Discord</a>
-      <a href="#">Connect</a>
+      <a class="header__button button button--secondary button--discord-icon" href="#">Discord</a> 
+      <a class="header__button button button--primary button--wallet-icon " href="#">Connect</a> 
     </header>
-    <main>
-      <section>
-        <article>
-          <h1>with <span>binabox</span> explore nft collection</h1>
-          <p>We are the best way to check rarity of NFT collection</p>
-          <dl>
-            <dt>2240+</dt>
-            <dd>Total iteam</dd>
-            <dt>1000+</dt>
-            <dd>Profiles whitelisted</dd>
+    <main class="main">
+      <section class="banner">
+        <article class="banner__article" >
+          <h1 class="banner__title">with <span>binabox</span> explore nft collection</h1>
+          <p class="banner__text">We are the best way to check rarity of NFT collection</p>
+          <dl class="banner__stats"> 
+            <dt class="banner__stats-term">2240+</dt>
+            <dd class="banner__stats-desc">Total iteam</dd>
+            <dt class="banner__stats-term">1000+</dt>
+            <dd class="banner__stats-desc">Profiles whitelisted</dd>
           </dl>
-          <a href="#">Connect wallet</a>
-          <a href="#">Whitelist now</a>
+          <a class="banner__button banner__button--primary button" href="#">Connect wallet</a> block here
+          <a class="banner__button banner__button--secondary button" href="#">Whitelist now</a>
         </article>
-        <div>
           <img src="#" alt="Main-Image_Cube" />
-        </div>
       </section>
-      <section>
-        <h2>Catalog</h2>
-        <a href="#">See all</a>
-        <button>&lt;</button>
-        <button>&gt;</button>
-        <ul>
-          <li>
-            <a href="#">
-              <article>
-                <span>New</span>
-                <img src="#" alt="Card - Skeleton" />
-                <p>$199</p>
-                <p>Name product</p>
+      <section class="catalog">
+        <h2 class="catalog__title">Catalog</h2>
+        <a class="catalog__button" href="#">See all</a>
+        <button class="catalog__button-left catalog__button-left--secondary">&lt;</button> 
+        <button class="catalog__button-right catalog__button-right--primary">&gt;</button>
+        <ul class="catalog__list">
+          <li class="catalog__list-item">
+            <a href="#"> 
+              <article class="prdouct-card">  
```

Comment:
You're absolutely right! And remember, you can always do section and article as an independent BEM Block.